packages/accounts/src/light-account/accounts/base.ts

```diff
@@ -242,7 +231,7 @@ export async function createLightAccountBase({
         case "v1.1.0":
           return signWith1271WrapperV1(hashMessage(message));
```

Comment:
Should this be `await`ed too?

Reply:
`return x` and `return await x` do the same thing (unless in a try-block), so the linter would probably complain at me.